src/clients/Lighthouse.ts

```diff
@@ -0,0 +1,32 @@
+import { AppClient, IOContext, InstanceOptions } from '@vtex/api'
+import { stringify } from 'querystring'
+
+const LH_TIMEOUT = 1e5
+
+export class Lighthouse extends AppClient {
+  constructor(ioContext: IOContext, opts?: InstanceOptions) {
+    super('vtex.lighthouse@0.x', ioContext, {
+      ...opts,
+      timeout: LH_TIMEOUT,
+    })
+  }
+
+  public runAudit(url: string): Promise<any[]> {
+    return this.http.post('/_v/toolbelt/audit/url', { url })
+  }
+
+  public getReports(app: string, url: string): Promise<any[]> {
```

Comment:
Instead of `any[]` can you define a type? Since it's saved in masterdata I imagine that this is well structured data and that you know all the fields that comes with it. Sorry if I'm wrong and `any` is the way to go

Reply:
Nope, there is a structured data for that. I'll put that on the code!